src/time_zone_fixed.cc

```diff
@@ -48,9 +48,11 @@ int Parse02d(const char* p) {
 }  // namespace
 
 bool FixedOffsetFromName(const std::string& name, seconds* offset) {
-  if (name.compare(0, std::string::npos, "UTC", 3) == 0) {
-    *offset = seconds::zero();
-    return true;
+  if (name.rfind("UTC", 0, 3) == 0) {
```

Comment:
What do you think about simply: `if (name == "UTC" || name == "UTC0")`?

Reply:
Sure.